weblogic/datadog_checks/weblogic/data/metrics.yaml

```diff
@@ -119,6 +119,9 @@ jmx_metrics:
         ExecutionTimeHigh:
           metric_type: monotonic_count
           alias: weblogic.servlet_runtime.exec_time_high
+        ExecutionTimeTotal:
+          metric_type: monotonic_count
```

Comment:
The correct metric type is documented in metadata.csv (`rate`) https://docs.datadoghq.com/metrics/types/?tab=count#submission-types-and-datadog-in-app-types